shenanigan/models/stackgan/run.py

```diff
@@ -103,6 +104,16 @@ def run(train_loader, val_loader, small_image_dims, results_dir, settings, stage
         plotter = LogPlotter(results_dir)
         plotter.learning_curve()
 
+    if evaluate:
+        eval_fxn(
+            model=model,
+            dataloader=val_loader,
+            experiment_name=experiment_name,
+            num_samples=settings['stage2']['num_samples'],
+            augment=True,
+            noise_size=settings['stage1']['noise_size']
+        )
```

Comment:
This looks like it is meant to be used for evaluating the full model. Perhaps we need to block this if a stage 1 model is selected in the CLI.

Reply:
good point, will make the change